public/sw.js

```diff
@@ -12,7 +12,7 @@ self.addEventListener('install', event => {
 
 const fetchEventHandler = async event => {
   if (
-    /^https:\/\/ichef(\.test)?\.bbci\.co\.uk\/(news|ace\/(standard|ws))\/.+(\.jpg|\.png)$/.test(
+    /^https:\/\/ichef(\.test)?\.bbci\.co\.uk\/(news|ace\/(standard|ws))\/.+\.webp$/.test(
```

Comment:
Let's not forget to update the service worker tests to cater for this new scenario (hopefully they are already failing!)

Reply:
No service worker tests are failing!

Reply:
errrr they are now and I don't know why they didn't before some small changes I made which were mostly code improvements...

Reply:
worked out why. Have changed tests. 
The logic change means the tests had to be changed or the URLs used wouldn't go into the condition of the sw that then does things. The tests were jpgs, so didn't match the new regex of expecting a webp url.